app/com/arpnetworking/metrics/portal/version_specifications/VersionSpecificationRepository.java

```diff
@@ -34,6 +34,16 @@
 public interface VersionSpecificationRepository {
```

Comment:
Not a fan of the underscore package name.

http://stackoverflow.com/questions/8243316/java-code-convention-for-package-names

I'm fine with scoping this under version or versions depending on plurality of its siblings.